crates/sim2h/src/cache.rs

```diff
@@ -19,7 +19,7 @@ pub struct Space {
     crypto: Box<dyn CryptoSystem>,
     agents: HashMap<AgentId, AgentInfo>,
     all_aspects_hashes: AspectList,
-    missing_aspects: HashMap<AgentId, HashSet<AspectHash>>,
+    missing_aspects: HashMap<AgentId, HashMap<EntryHash, Vec<AspectHash>>>,
```

Comment:
Can this be `HashSet<AspectHash>` rather than `Vec<AspectHash>` to make it explicit there can be no duplicates?

Reply:
:+1: